packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -82,6 +82,8 @@ class MobileAdTargetingInfo {
   }
 }
 
+enum AnchorType { anchorBottom, anchorTop }
```

Comment:
Could be simplified to `enum AnchorType { bottom, top }` since the `anchor` prefix is already part of the enum type.